packages/tools/src/cmd/autofix.cmd.ts

```diff
@@ -185,3 +186,52 @@ export const autofixBuildCmd = new Command('build-container')
 		await fs.writeFile(wranglerPath, updatedContent)
 		echo(chalk.green(`\nUpdated wrangler.jsonc with new image: ${registryImage}`))
 	})
+
+export const autofixFixBuildCmd = new Command('fix-build')
```

Comment:
got tired of the manual curl's. This can be used with `just autofix fix-build <uuid>`